etc/notebooks/examples/ArxivBokeh.ipynb

```diff
@@ -31,9 +31,9 @@
    },
    "outputs": [],
    "source": [
-    "from urth import widgets\n",
+    "from declarativewidgets import init\n",
```

Comment:
Minor comment: I'd prefer to see consistency in how we invoke init across our example notebooks. I see the following being used:

```
import declarativewidgets as widgets
widgets.init()
```

```
from declarativewidgets import init
init()
```

```
import declarativewidgets as declwidgets
declwidgets.init()
```

```
import declarativewidgets
declarativewidgets.init()
```